frontend/tests/controllers-test/challengeCtrl.test.js

```
@@ -1646,8 +1640,7 @@ describe('Unit tests for challenge controller', function () {
             expect(vm.isResult).toEqual(true);
             expect(vm.phaseId).toEqual(phaseId);
 
-            expect(utilities.storeData).toHaveBeenCalledWith('emailError', errorResponse.detail);
-            expect($state.go).toHaveBeenCalledWith('web.permission-denied');
+            expect(utilities.handlePermissionDeniedError).toHaveBeenCalled();
```

ESLint

[error] 'expect' is not defined.

(no-undef)